crates/web-client/test/mocha.global.setup.mjs

```
@@ -26,10 +26,10 @@ before(async () => {
   console.log("Starting test server...");
   serverProcess = spawn("http-server", ["./dist", "-p", TEST_SERVER_PORT], {
     stdio: "inherit",
-    shell: process.platform == 'win32'
+    shell: process.platform == "win32",
   });
 
-  browser = await puppeteer.launch({ headless: true });
+  browser = await puppeteer.launch({ headless: true, protocolTimeout: 600000 });
```

[The default timeout is 180000](https://pptr.dev/api/puppeteer.browserconnectoptions#properties), do we need to increase it? I think this is why the CI tests sometimes took more than an hour to fail as some test errors just end up in a timeout. We could even reduce it if 180000 is too much so that the tests fail faster.